composer.json

```diff
@@ -20,10 +20,7 @@
         "jakub-onderka/php-console-highlighter": "0.3.*"
     },
     "require-dev": {
-        "phpunit/phpunit": "~3.7|~4.0",
-        "symfony/finder": "~2.1",
```

Comment:
Yeah, but since I'm using it, I'd prefer not to rely on a dependency of a dependency, since that's not guaranteed to remain the case.